### DISCUSSION TITLE
Would like to help you by becoming maintainer for your project

### DISCUSSION THREAD

**Discussion by CandidDeer** [anuraghazra/github-readme-stats#1976] [Q&A] [answered]
Hi there, I'd like to help you clear up the backlog of issues and PRs. If you let me be a maintainer for your project, I'd love to help you out, no strings attached. ✨

**Comment by rickstaa** [comment, marked as answer]
@CandidDeer First of all, welcome to the community! Great to hear that you want to contribute. 🚀  All the help is more than welcome. 😃 On top of my head, there are several things you can do:
- Answer questions in the [discussions](https://github.com/anuraghazra/github-readme-stats/discussions).
- Fix [bugs](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
- Implement [new features](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
- Improve the documentation.

Please check out the [contribution](https://github.com/anuraghazra/github-readme-stats/blob/master/CONTRIBUTING.md) guidelines on how to get started. You can check #1935 to see the most requested features or pressing bugs and choose a feature bug you want to work on. Alternatively, you can check the [help-wanted](https://github.com/anuraghazra/github-readme-stats/labels/help%20wanted) label or, of course, also create a new issue if you have a new idea on how we can improve GHS. I or @anuraghazra will look at your issue and assign you to it. Please don't hesitate to ask questions if anything is unclear.

**Reply by CandidDeer** [reply]
Thank you, @rickstaa. Would it be possible to give me write access to the repo, though, so I can merge PRs, too? Otherwise I will only be able to contribute to the PR backlog. 😉

**Reply by rickstaa** [reply]
@CandidDeer That decision is ultimately up to @anuraghazra. However, I think it is better for people to first contribute to the repository and show they are invested before making collaborators. I hope you find something interesting to work on. Don't hesitate to ask other questions.

**Reply by rickstaa** [reply]
@CandidDeer I just thought about it a bit more, and I think implementing #1659 would have the most effect on the PR backlog since it would allow us to close all the theme PRs that would not adhere to our guidelines. Another easy one that might filter out issues is #1950. However, feel free to implement or contribute in any way you like.
